guile/tests/meson.build

```diff
@@ -21,6 +21,7 @@
 if get_option('b_sanitize') == 'none'
   guile_load_path=':'.join([
     join_paths(meson.project_source_root(), 'guile'),
```

Comment:
Actually for the current status this line can be dropped, though not sure whether there will be future use of this, so I didn't do it yet.

Reply:
Hmm, `GUILE_LOAD_PATH` is for Scheme source files IIRC, and we don't have any in the build-dir.

Reply:
Ack, then it's better to leave this intact.